product_docs/docs/biganimal/release/using_cluster/pgd_cli_ba.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: PGD CLI on BigAnimal
+navTitle: PGD CLI on BigAnimal
+deepToC: true
+---
+
+When running a distributed high-availability cluster (DHA) on BigAnimal, you can use the [PGD CLI](../../../pgd/latest/cli/) to manage cluster operations, such as switching over write leaders, performing cluster health checks, and viewing various details about nodes, groups, or other aspects of the cluster.
+
+## Installing the PGD CLI 
+
+To [install the PGD CLI](../../../pgd/latest/cli/installing_cli/), replace `<your-token>` with your EDB subscription token in the following command for Debian and Ubuntu machines:
+
+```bash
+curl -1sLf 'https://downloads.enterprisedb.com/&lt;your-token&gt;/postgres_distributed/setup.deb.sh' | sudo -E bash
+sudo apt-get install edb-pgd5-cli
+```
+
+or in this command for RHEL, Rocky, AlmaLinux, or Oracle Linux machines:
+
+```bash
+curl -1sLf 'https://downloads.enterprisedb.com/&lt;your-token&gt;/postgres_distributed/setup.rpm.sh' | sudo -E bash
+sudo yum install edb-pgd5-cli
+```
+
+## Connecting to your BigAnimal cluster
+
+### Discovering your database connection string
+
+To connect to your DHA BigAnimal cluster via the PGD CLI, you need to [discover the database connection string](../../../pgd/latest/cli/discover_connections/) from your BigAnimal console:
+
+1. Log into the [BigAnimal clusters](https://portal.biganimal.com/clusters) view.
+1. In the filter, set **Cluster Type** to **Distributed High Availability** to show only clusters that work with PGD CLI.
+1. Select your cluster.
+1. In the view of your cluster, select the **Connect** tab.
+1. Copy the read/write URI from the connection info. This is your connection string.
+
+### Using the PGD CLI with your database connection string
+
+!!! Important
+PGD does not prompt for interactive passwords. Accordingly, you should have a [`.pgpass` file](https://www.postgresql.org/docs/current/libpq-pgpass.html) properly configured to allow access to the cluster. Your BigAnimal cluster's connection information page has all the necessary information needed for the file. 
+
+Without a properly configured `.pgpass`, you receive a database connection error when using a PGD CLI command, even when using the correct database connection string with the `--dsn` flag.
+!!!
+
+To use the PGD CLI with your database connection string, use the `--dsn` flag with your PGD CLI command:
+
+```bash
+pgd show-nodes --dsn "<your_connection_string>"
+```
+
+## PGD commands in BigAnimal
+
+!!! Note
+There are three EDB Postgres Distributed CLI commands that don't apply to BigAnimal: `create-proxy`, `delete-proxy` and `alter-proxy-option`.
```

Comment:
@jpe442 could we add a sentence about why they don't apply? because they're already created, BA is a managed service etc just so people have some sense of why they aren't allowed

Reply:
I put an ask out in #eng-pgd. Will ask DJ as well.

Reply:
Okay, basically the guidance is that we should say they don't apply to BA because BA manages those three functions, as BA runs on Kubernetes and those activities are handled by the operator according to technical best practices.